kubectl-plugin/kubectl-waiops

```diff
@@ -1575,6 +1607,16 @@ then
         MAJORVERSION_ASM="2.24"
     }
 
+    component-versions-47() {
+        MAJORVERSION_AIOPSUI="4.7"
+        MAJORVERSION_AIMANAGER="4.7"
+        MAJORVERSION_IRCORE="4.7"
+        MAJORVERSION_LIFECYCLESERVICE="4.7"
+        MAJORVERSION_AIOPSANALYTICSORCHESTRATOR="4.7"
+        MAJORVERSION_ASM="2.25"
+        MAJORVERSION_FLINKDEPLOYMENT="1.19.0-aiops"
```

Comment:
I think we can exclude Flink here. The 1.19 is the version of the Flink image we maintain internally, not the operator. We also don't include Elastic or other dependencies here.

Reply:
This would also mean removing the upgrade status section for Flink, which I think is fine